test/pylib/scylla_cluster_manager.py

```
@@ -215,145 +184,118 @@ class ScyllaClusterManager:
-                             self.cluster, self.test_uname)
-            await self.cluster.recycle()
+            self.logger.info("ScyllaManager: stopping Scylla cluster %s after %s", self.cluster, self.test_name)
+            await self.cluster.stop()
             self.cluster = None
-        self.is_running = False
 
-    @manager_op
-    async def is_dirty(self) -> bool:
-        """Report if current cluster is dirty."""
+    @asynccontextmanager
+    async def run_in_thread(self) -> AsyncIterator[ScyllaClusterManager]:
```

Ruff (0.16.3)

[error] Undefined name `ScyllaClusterManager`

(F821)